src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -257,7 +260,8 @@ public FormValidation doCheckTaCoberturaChkBx(@QueryParameter boolean taCobertur
         }
 
         Function<String, FormValidation> chkCoberturaSupport = (String matlabRoot) -> {
-            rel = new MatlabReleaseInfo(matlabRoot);
+            FilePath matlabRootPath = new FilePath(new File(matlabRoot));
```

Comment:
If matlabroot is always consumed by first creating a FilePath object, then it make sense to keep the FilePath object at the same scope as the matlabroot (or even replace it if needed).

Reply:
captured this as task. may have to call it using reflections if i change the scope of this outside of this static class. captured it. i will discuss the possible solutions during commits of the captured task for the same.